schema/url/plots.raml

```diff
@@ -161,14 +161,15 @@ types:
     properties:
       config: MosaicSpec
 
+  # TODO consider showMissingness here. for 2x2 if we allow axes vars it wont be 2x2 any longer...
+  # in practical terms that means the associated stats dont work. for now not allowing in either mosaic
```

Comment:
agreed with keeping it ShowMissingnessNoAxes. I don't think the stats would make sense in the RxC case. I suppose we could turn off stats like we do in some other no data cases but...